### PR TITLE
Re-performing fix for #21

### PR DESCRIPTION
**Pull request by SphtKr**
Superceeds #26

### CODE REVIEW

**Comment by SphtKr** [comment]
@csantero Don't know if you got to look at #26, but this is the same thing redone against master...should be easier to read.

File: JSONAPI/Json/JsonApiFormatter.cs [hunk 1]
```diff
@@ -180,6 +180,13 @@ protected void Serialize(object value, Stream writeStream, JsonWriter writer, Js
         {
             writer.WriteStartObject();
 
+            //FIXME: The spec no longer requires that the ID key be "id":
```
Comment (SphtKr) [inline JSONAPI/Json/JsonApiFormatter.cs]: This in particular I'm unsure of. At the very least if we serialize e.g. `Uuid` as `id`, we should also include it as `uuid`, which it doesn't do yet. Which way to go here?

Comment (csantero) [inline JSONAPI/Json/JsonApiFormatter.cs]: I think that any property that is being used as the id property should be serialized as "id" only.

**Comment by csantero** [comment]
LGTM

File: JSONAPI/Json/JsonApiFormatter.cs (hunk 1 above)
Comment (SphtKr) [inline JSONAPI/Json/JsonApiFormatter.cs]: The more I think about it, to stick to the spec we probably shouldn't serialize it as "id" at all if that's not its name...but no option is clearly more appealing. I'm going to write both for now, but we'll probably revisit this, maybe the spec will change...again. (I liked it better when it simply required your ID to be "id".)

Comment (csantero) [inline JSONAPI/Json/JsonApiFormatter.cs]: When did it change? I thought "ID" was still expected.

**Comment by SphtKr** [comment]
Closes #21

**Pull request merged**